packages/devtools_app/test/test_infra/scenes/standalone_ui/utils.dart

```diff
@@ -0,0 +1,45 @@
+// Copyright 2024 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be found
+// in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:stream_channel/stream_channel.dart';
+import 'package:web_socket_channel/web_socket_channel.dart';
+
+/// Connects to the websocket at [wsUri] and returns a channel along with
+/// a log stream that includes all protocol traffic.
+Future<({StreamChannel<String> channel, Stream<String> log})>
+    createLoggedWebSocketChannel(Uri wsUri) async {
```

Comment:
nit: trailing comma

Reply:
Still looked naff with a trailing comma, so I pulled the return type out into a typedef to avoid the wrapping in d50c807fe